ui/app/src/components/NewExperimentNext/Step1.tsx

```diff
@@ -60,6 +61,33 @@ const useStyles = makeStyles((theme) => {
 
 const submitDirectly = ['pod-failure']
 
+interface TypeCardProp {
+  name: Env
+  handleSwitchEnv: (env: Env) => () => void
+  env: Env
+}
+
+const TypeCard: React.FC<TypeCardProp> = ({ name, handleSwitchEnv, env }) => {
+  const classes = useStyles()
+  const title = name === 'k8s' ? 'k8s.title' : 'physics.single'
+  return (
+    <Card
+      className={clsx(classes.card, env === name ? classes.cardActive : '')}
+      variant="outlined"
+      onClick={handleSwitchEnv(name)}
+    >
+      <Box display="flex" justifyContent="center" alignItems="center" width={225} height={75}>
+        <Box display="flex" justifyContent="center" flex={1}>
+          {iconByKind(name)}
+        </Box>
+        <Box flex={1.5} textAlign="center">
+          <Typography variant="button">{T(`${title}`)}</Typography>
```

Comment:
T(`${title}`) is equal with `T(title)`. It's better to pass the `title` directly.

Reply:
Fixed.